docs/source/get_started/access_info/access_info_css.ipynb

```diff
@@ -84,7 +84,7 @@
     {
```

Comment:
<p>Technically unrelated to PR, but worth re-running the <code>get_balance()</code> cells by merging <a href="https://github.com/Infleqtion/client-superstaq/pull/815" target="_blank">https://github.com/Infleqtion/client-superstaq/pull/815</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/728/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Merged with [fab0cdc](https://github.com/Infleqtion/client-superstaq/pull/728/commits/fab0cdc2c1caa30041581f0ff2b3d06c417df237)